UPGRADE.md

```diff
@@ -1,4 +1,24 @@
 # UPGRADING
+## Recommended way of upgrading
+* upgrade locally first. After you fix all issues caused by the upgrade, version them and then continue with upgrading application on a server
+* upgrade one version at a time:
+    * Start with a working application
+    * Upgrade to the next version
+    * Fix all issues
+    * Repeat
+* typical upgrade sequence should be:
+    * `docker-compose down`
+    * follow upgrade notes for `docker-compose.yml`, `Dockerfile`, docker containers
+    * `docker-compose up -d`
+    * update shopsys framework dependencis in `composer.json` to version you are upgrading to
+    * `composer update`
+    * follow all upgrade notes you didn't do yet
+    * `php phing clean`
+    * `php phing db-migrations`
+    * version these changes
+* even we care a lot about these instructions, it is possible we miss something. In case something doesn't work after the upgrade,
+  you'll find more information in the [CHANGELOG](CHANGELOG.md)
```

Comment:
why the line break?

Reply:
just because the line would be too long (won't fit on my screen)

Reply:
I am ok with it here for now but for the future cases, I recommend using soft wraps settings in PHPStorm (my shortcut is `ctrl` + `shift` + `w`)